src/test/librbd/test_mock_ManagedLock.cc

```diff
@@ -393,6 +393,27 @@ TEST_F(TestMockManagedLock, AcquireLockBlocklist) {
   ASSERT_EQ(0, when_shut_down(managed_lock));
 }
 
+TEST_F(TestMockManagedLock, SendAcquireLockBlocklist) {
```

Comment:
For consistency with other `AcquireLock` tests which all go through `send_acquire_lock`:
```suggestion
TEST_F(TestMockManagedLock, AcquireLockBlocklistedWatch) {
```